FileManager/AppDelegate.swift

```diff
@@ -10,27 +10,27 @@ import UIKit
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
+  lazy var window: UIWindow? = UIWindow(frame: UIScreen.main.bounds)
```

Comment:
Тут такое дело, я по привычке снес сториборд и зачем делегат сцены, хотя нынче надо это делать всё именно там. Так вот если UIWindow объявить просто var - вылезает в консоли вот такая беда "[ApplicationLifecycle] UIWindows were created prior to initial application activation. This may result in incorrect visual appearance.". Так как в пылу мне уже стало лень заморачиваться с восстановлением SceneDelegate.swift - нашел вот такое решение (stackoverflow) работает теперь как надо и без ворнингов. Надо ли в итоге entry point  в делегат сцены переносить?